src/other/Table.js

```diff
@@ -30,7 +30,7 @@
     Table.prototype.publish("fixedHeader", false, "boolean", "Enable or disable fixed table header",null,{tags:["Private"]});
     Table.prototype.publish("fixedColumn", false, "boolean", "Enable or disable fixed first column",null,{tags:["Private"]});
     
-    Table.prototype.publish("fixedSize", false, "boolean", "Fix Size to Min Width/Height");
+    Table.prototype.publish("fixedSize", true, "boolean", "Fix Size to Min Width/Height");
```

Comment:
It should default **false** (allowing 100% width and 100% height by default). The **tableDiv** is what this publish param ultimately controls, which should usually adapt to its parent's dimensions.